PillarChap/Pillar.pillar

```diff
@@ -113,18 +125,27 @@ This should generate a ==first.html== file you can open in a web browser. This c
 
 !!!Configuring a document
 
-As you can see, there is no title in the generated ==first.html== file. This is because we did not specify any. To specify a title, we have to add it as a 
-meta-information at the beginning of the ==first.pillar== file:
+As you can see, there is no title in the generated ''output.html'' file. This is because we did not specify any. To specify a title, we have to add it with a configuration
+at the beginning of the ''output.html'' file:
```

Comment:
The file name is output.html because by default it export into the same output file and we do not have a «--outputFile=First.html» parameter.